src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2105,6 +2105,7 @@ internal enum ErrorCode
         ERR_FileTypeNested = 9054,
         ERR_GlobalUsingStaticFileType = 9055,
         ERR_FileTypeNameDisallowed = 9056,
+        ERR_FeatureNotAvailableInVersion11 = 9057,
```

Comment:
>9057 [](http://example.com/codeflow?start=45&length=4)

This id is used in [https://github.com/dotnet/roslyn/pull/61261](https://github.com/dotnet/roslyn/pull/61261).

Reply:
I think we can adjust the numbers before shipping. The linked PR above is going into 17.3 so I think if we just flow it naturally then the error ID in this PR will adjust.

Reply:
@RikkiGibson I shifted the new error code by one to avoid conflict with your 17.3 change (make merging easier for Tiger).
@cston Thanks for pointing this out.